{"uid":"8217f65e85617d3c","name":"test_preflight_check","fullName":"tests.integration.upgrade.test_upgrade#test_preflight_check","historyId":"10ce8e88c2e84b3f733e7d5fd062a049","time":{"start":1722563652650,"stop":1722563653789,"duration":1139},"description":"Verifies that the preflight check can run successfully.","descriptionHtml":"<p>Verifies that the preflight check can run successfully.</p>\n","status":"failed","statusMessage":"AssertionError: pre-upgrade-check failed, expected to succeed.\nassert 'failed' == 'completed'\n  \n  - completed\n  + failed","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/tests/integration/upgrade/test_upgrade.py\", line 85, in test_preflight_check\n    assert action.status == \"completed\", \"pre-upgrade-check failed, expected to succeed.\"\nAssertionError: pre-upgrade-check failed, expected to succeed.\nassert 'failed' == 'completed'\n  \n  - completed\n  + failed","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1722562650549,"stop":1722562650550,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1722562650550,"stop":1722562650890,"duration":340},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1722562650550,"stop":1722562650550,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1722563652650,"stop":1722563652650,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Verifies that the preflight check can run successfully.","status":"failed","statusMessage":"AssertionError: pre-upgrade-check failed, expected to succeed.\nassert 'failed' == 'completed'\n  \n  - completed\n  + failed","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/mongodb-operator/mongodb-operator/tests/integration/upgrade/test_upgrade.py\", line 85, in test_preflight_check\n    assert action.status == \"completed\", \"pre-upgrade-check failed, expected to succeed.\"\nAssertionError: pre-upgrade-check failed, expected to succeed.\nassert 'failed' == 'completed'\n  \n  - completed\n  + failed","steps":[],"attachments":[{"uid":"1aa9adee5bae253d","name":"log","source":"1aa9adee5bae253d.txt","type":"text/plain","size":95}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1722563653791,"stop":1722563653791,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1722564657305,"stop":1722564657984,"duration":679},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1722564657985,"stop":1722564657986,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1722564657987,"stop":1722564657987,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1722564657987,"stop":1722564657988,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.upgrade"},{"name":"suite","value":"test_upgrade"},{"name":"host","value":"large-457-361f67e8fc52e67f09f36797"},{"name":"thread","value":"10561-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.upgrade.test_upgrade"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"598eb9b72609b3b1","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"7ffe91ded53e69c5","reportUrl":"../2012//#testresult/7ffe91ded53e69c5","status":"passed","time":{"start":1722482299224,"stop":1722482422508,"duration":123284}},{"uid":"71c574076cf2d37","reportUrl":"../2006//#testresult/71c574076cf2d37","status":"passed","time":{"start":1722394157454,"stop":1722394281962,"duration":124508}},{"uid":"661f377242c75821","reportUrl":"../2005//#testresult/661f377242c75821","status":"passed","time":{"start":1722306333815,"stop":1722306577371,"duration":243556}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","group(1)"]},"source":"8217f65e85617d3c.json","parameterValues":[]}